### PR TITLE
pak::pak() tweaks

### PR DESCRIPTION
**Pull request by hadley**
A couple of minor behaviour tweaks

### CODE REVIEW

File: R/local.R [hunk 1]
```diff
@@ -74,6 +74,8 @@ local_install_deps <- function(root = ".", lib = .libPaths()[1],
 
 local_install_dev_deps <- function(root = ".", lib = .libPaths()[1],
                                    upgrade = TRUE, ask = interactive()) {
+
+  root <- rprojroot::find_package_root_file(path = root)
```
Comment (gaborcsardi) [inline R/local.R:78]: Yeah, this would be good, but we need to load packages in the subprocess.

File: R/pak.R [hunk 2]
```diff
@@ -14,18 +14,20 @@
...
+#' @inheritParams pkg_install
 #' @param ... Extra arguments are passed to [pkg_install()] or
 #'   [local_install_dev_deps()].
 #'
 #' @export
 #' @family package functions
 #' @family local package trees
 
-pak <- function(pkg = NULL, ...) {
+pak <- function(pkg = NULL, upgrade = FALSE, ...) {
```
Comment (gaborcsardi) [inline R/pak.R:27]: I actually omitted this on purpose, because the default is different now for `pkg_install()` and `local_install_dev_deps()`.

**Comment by hadley** [comment]
Bummer. Probably best to just close this then.

**Comment by gaborcsardi** [comment]
We could put some minimal rprojroot into pak. Or just use it in the subprocess, that is not any slower in practice.

**Comment by hadley** [comment]
The main thing is to do it before the metadata download/processing

**Comment by gaborcsardi** [comment]
Yeah, so why not just do it as first thing in the subprocess? We should probably do it for all `local_*()` functions.

**Comment by gaborcsardi** [comment]
@hadley I can update this PR.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**